specification/cognitiveservices/data-plane/Personalizer/preview/v2022-09-01-preview/Personalizer.json

```diff
@@ -0,0 +1,2847 @@
+					"format": "duration",
+					"description": "Aggregation window.",
+					"type": "string",
+					"readOnly": true
+				},
+				"value": {
+					"description": "Time series of the learning settings results.",
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/PolicyResult"
+					}
+				}
+			}
+		},
+		"Policy": {
+			"type": "object",
+			"properties": {
+				"name": {
+					"description": "The name of the Learning Settings.",
+					"type": "string",
+					"readOnly": true
+				},
+				"arguments": {
+					"description": "The arguments of the Learning Settings.",
+					"type": "string",
+					"readOnly": true
+				},
+				"policySource": {
+					"description": "The source of the Learning Settings.",
+					"enum": [
+						"Online",
+						"Baseline",
+						"Random",
+						"Custom",
+						"OfflineExperimentation"
+					],
+					"type": "string",
+					"readOnly": true,
+					"x-ms-enum": {
+						"name": "PolicySource",
+						"modelAsString": true
+					}
+				}
+			}
+		},
+		"Evaluation": {
+			"description": "A counterfactual evaluation.",
+			"type": "object",
+			"properties": {
+				"id": {
+					"description": "The ID of the evaluation.",
+					"maxLength": 256,
+					"type": "string",
+					"readOnly": true
+				},
+				"name": {
+					"description": "The name of the evaluation.",
+					"maxLength": 256,
+					"type": "string",
+					"readOnly": true
+				},
+				"startTime": {
+					"format": "date-time",
+					"description": "The start time of the evaluation.",
+					"type": "string",
+					"readOnly": true
+				},
+				"endTime": {
+					"format": "date-time",
+					"description": "The end time of the evaluation.",
+					"type": "string",
+					"readOnly": true
+				},
+				"creationTime": {
+					"format": "date-time",
+					"description": "The creation time of the evaluation.",
+					"type": "string",
+					"readOnly": true
+				},
+				"status": {
+					"description": "The status of the job processing the evaluation.",
+					"enum": [
+						"completed",
+						"pending",
+						"failed",
+						"notSubmitted",
+						"timeout",
+						"optimalPolicyApplied",
+						"onlinePolicyRetained"
+					],
+					"type": "string",
+					"readOnly": true,
+					"x-ms-enum": {
+						"name": "EvaluationJobStatus",
+						"modelAsString": true
+					}
+				},
+				"evaluationType": {
+					"description": "The type of evaluation.",
+					"enum": [
+						"Manual",
+						"Auto"
+					],
+					"type": "string",
+					"readOnly": true,
+					"x-ms-enum": {
+						"name": "EvaluationType",
+						"modelAsString": true
+					}
+				},
+				"optimalPolicy": {
+					"description": "Optimal policy found by the evaluation job.",
+					"type": "string",
+					"readOnly": true
+				},
+				"content": {
+					"$ref": "#/definitions/EvaluationResult"
+				},
+				"description": {
+					"description": "Description of the evaluation job.",
+					"type": "string",
+					"readOnly": true
+				},
+				"policyResults": {
```

Comment:
It is a little odd that this property is named "policyResults" but is an array of "Policy" and not an array of "PolicyResults". Perhaps this property should be named "policies"?

Reply:
Delete not required property.